article/ttp-tutorial.html

```diff
@@ -230,45 +240,57 @@ <h3 class="card-title text-uppercase text-center text-white" style="font-weight:
         <div id="paperSubmission" class= "tab-pane" role="tabpanel">
           <div class="row pt-5">
             <div class="col-12">
-              <p>This tutorial will guide you through the submission of your paper.</p>
-              <p>1. Log in to your account.</p>
-              <img src="../images/tutorials/ttp-paper-submission-tutorial/1.png" class="rounded mb-3 img-tutorial">
-              <p>2. Once you are logged in, click <code>Participant</code> on the top menu. On the participant home tab download the doc or "LaTex" template to prepare the manuscript in accordance with the publisher’s technical requirements.</p>
-              <img src="../images/tutorials/ttp-paper-submission-tutorial/2.png" class="rounded mb-3 img-tutorial">
-              <p>3. Click the upload tab to start the submission process. Complete the six stages listed on the left-side menu.</p>
+              <div class="alert alert-danger alert-dismissible fade show" role="alert">
+                <ul class="fa-ul" style="margin-bottom: 0rem;">
+                  <li>
+                    <i class="fa fa-exclamation-triangle fa-li" aria-hidden="true"></i>All the paper’s author must <b>register a TTP account</b> before submitting your paper. 
+                  </li>
+                  <li>
+                    <i class="fa fa fa-exclamation-circle fa-li" aria-hidden="true"></i>Please ensure that you are using the correct account to submit your paper, as an invoice will be generated for this account.
```

Comment:
Double `fa`.

This one is using `fa-exclamation-circle`, but the previous one is using `fa-exclamation-triangle`. Is this on purpose?